src/plugins/unified_search/public/data_view_select/create_data_view_select.tsx

```diff
@@ -8,12 +8,12 @@
 
 import React from 'react';
 
-import { IndexPatternsContract } from '@kbn/data-plugin/public';
-import { IndexPatternSelect, IndexPatternSelectProps } from '.';
+import type { DataViewsContract } from '@kbn/data-plugin/public';
```

Comment:
Let's fetch the contract from the dataViews plugin `'@kbn/data-views-plugin/public'`

Reply:
Yes, sure. You are right. I'll check where else you can change the imports from `@kbn/data-plugin/public` to `@kbn/data-views-plugin/public`